packages/notebook/src/actions.tsx

```diff
@@ -1,8 +1,6 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
```

Comment:
There's a lot of changes here in notebook actions. We will have to test it carefully, as there are lots of subtle focus, selection, and markdown rendering issues.